src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574660,31 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1030003
+name: Pallister-Hall-like syndrome
+def: "A pleiotropic autosomal recessive disorder characterized by phenotypic variability. Patients exhibit postaxial polydactyly as well as hypothalamic hamartoma, cardiac and skeletal anomalies, and craniofacial dysmorphisms. Hirschsprung disease has also been observed." [OMIM:241800, PMID:30497210, PMID:32413283]
+synonym: "PHLS" EXACT ABBREVIATION [OMIM:241800]
+xref: MEDGEN:137970 {source="https://orcid.org/0000-0002-3458-4839"}
+xref: OMIM:241800 {source="MONDO:equivalentTo", source="https://orcid.org/0000-0002-3458-4839"}
+is_a: MONDO:1030005 {source="https://clinicalgenome.org/affiliation/40059/"} ! mosaic SMO syndrome
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11119 ! SMO
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7768" xsd:anyURI
+
+[Term]
+id: MONDO:1030005
+name: mosaic SMO syndrome
+def: "A syndrome characterized by cutaneous and skeletal manifestations such as linear hypo‐ or hyperpigmented lesions, basaloid follicular hamartomas, palmoplantar pitting, atrophoderma, hypertrichosis, polydactyly or syndactyly, rib anomalies (rudimentary ribs), and limb‐length anomalies." [https://clinicalgenome.org/affiliation/40059/]
+xref: PMID:27236920 {source="https://clinicalgenome.org/affiliation/40059/"}
+xref: PMID:30497210 {source="https://clinicalgenome.org/affiliation/40059/"}
+xref: PMID:32413283 {source="https://clinicalgenome.org/affiliation/40059/"}
+xref: PMID:7606318 {source="https://clinicalgenome.org/affiliation/40059/"}
+is_a: MONDO:0015160 {source="https://orcid.org/0000-0002-3458-4839"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11119 {source="https://clinicalgenome.org/affiliation/40059/"} ! SMO
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/pull/7879" xsd:string
```

Comment:
this should be "xsd:anyURI" instead of "xsd:string"

Reply:
fixed

Reply:
also updated to link to issue vs. PR